.github/workflows/flink-ci.yml

```diff
@@ -66,6 +66,7 @@ jobs:
       matrix:
         jvm: [8, 11]
         flink: ['1.12', '1.13', '1.14']
+        scala: ['2.11', '2.12']
```

Comment:
This makes Flink testing very expensive. Is it possible to just test the latest Flink with all versions of Scala and test the old ones with just 2.12?

Reply:
+1. This will add quite a lot to the Flink CI time.

Reply:
Sounds reasonable ! Just updated the patch.